### PR TITLE
added api calls for timers

### PR DESCRIPTION
**Pull request by jenjenp**

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -2236,6 +2238,131 @@ paths:
...
+        - in: path
+          name: service_id
+          description: Environment to use
+          type: string
+          required: true
+        - in: body
+          name: timer
+          description: Single timer
+          schema:
+            $ref: "#/definitions/Timer"
```
Comment (chook) [inline services.yaml:2279]: only problem i have with this is that the Timer object already has an ID

File: services.yaml [hunk 2]
```diff
@@ -2576,6 +2703,34 @@ definitions:
...
+      threshold:
+        type: integer
+        description: Threshold in ms
+        example: 8000
+      servers:
+        type: array
+        items:
+          type: string
+          example: na-dotCMS-2
+      applications:
```
Comment (chook) [inline services.yaml:2729]: we should support deployments as well

File: services.yaml (hunk 1 above)
Comment (jenjenp) [reply services.yaml:2279]: following our conversation - this is the same approach that we use for Views, categories, etc.. we ignore the 'id' that user defines.

File: services.yaml (hunk 2 above)
Comment (jenjenp) [reply services.yaml:2729]: at first stage decided to align with the UI which is missing the deployment

**Pull request merged**